config/initializers/devise.rb

```diff
@@ -1,9 +1,25 @@
 # frozen_string_literal: true
 
+def cert_path
+  cert_path = Rails.root.join('tmp', 'omniauth-cert.p12')
+  cert_data = ENV.fetch('ENTRA_CERTIFICATE_DATA', nil)
+
+  return cert_path if File.exist?(cert_path)
+  return nil if cert_data.blank?
+
+  cert_data = Base64.strict_decode64(cert_data)
+
+  File.binwrite(cert_path, cert_data)
+  File.chmod(0o600, cert_path)
```

Comment:
assuming the intention here is to set the file rights to: Owner can: Read, Write

What's the "0o" in the beginning here? Doesn't it just need to be 600?

Reply:
It's 600 in octal notation not the decimal number 600